core/server/master/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -331,7 +331,7 @@ void rename(AlluxioURI srcPath, AlluxioURI dstPath, RenameOptions options)
   // into RuntimeException on the client.
   void free(AlluxioURI path, FreeOptions options)
       throws FileDoesNotExistException, InvalidPathException, AccessControlException,
-      UnexpectedAlluxioException, UnavailableException;
+      UnexpectedAlluxioException, IOException;
```

Comment:
Why is this changed?

Reply:
Because we now use  `Closer` to close resources, which can throw an `IOException`